vendor/Comment.php

```diff
@@ -0,0 +1,19 @@
+<?php
+include "User.php";
+class Comment
+{
+    private $user, $message;
```

Comment:
По PSR каждое поле класса должно быть объявлено отдельно:
```suggestion
    private User $user;
    private string $message;
```